src/constants.js

```diff
@@ -379,6 +433,11 @@ export const styles = {
     lineHeight: '50px'
   },
 
+  statementSmall: {
+    fontSize: 18,
+    paddingBottom: 6
+  },
+
   statementLabel: {
```

Comment:
I know this is a small thing but can we round the corners on each word and give a little more left/right padding?

Reply:
Is there any concern that it makes these look like buttons?

![Screen Shot 2021-04-29 at 11 13 24 AM](https://user-images.githubusercontent.com/2205926/116491179-236a0c00-a84e-11eb-98af-c791adbda1d0.png)

Reply:
+1 to the concern that rounded corners make them look like buttons, also the square corners more closely match the column headings and anything we can do to reinforce that connection (e.g. like we did with color) feels like a good direction to head

Reply:
I like the rounded corners. It matches the rounded corners of the model card. I think it's okay for the features to look like buttons in the mad lib since you can click on them to remove them.